cfgmassif.go

```diff
@@ -39,13 +39,14 @@ func cfgMassifReader(cmd *CmdCtx, cCtx *cli.Context) error {
 	}
 
 	if localLog == "" && remoteLog == "" {
-		// if we had no url and no local data supplied we try STDIN
-		opener := cfgStdinOpener()
-		mr, err := NewLocalMassifReader(logger.Sugar, opener, "-")
+		// If we had no url and no local data supplied we default to the production data source.
+		reader, err := cfgReader(cmd, cCtx, true)
 		if err != nil {
 			return err
 		}
-		cmd.massifReader = mr
+		mr := massifs.NewMassifReader(logger.Sugar, reader, massifs.WithoutGetRootSupport())
```

Comment:
minor: do you really mean "WithoutGetRootSupport()". If it is not set, the peak stack is not "mapped" and attempts to randomly access it will fail when generating proofs of inclusion. Unless you specifically want to operate exactly as we do when adding leaves to the mmr, it is best to ommit this option